config/set/php80.php

```diff
@@ -45,6 +46,7 @@
         ClassOnThisVariableObjectRector::class,
         ConsistentImplodeRector::class,
         OptionalParametersAfterRequiredRector::class,
+        MbStrContainsRector::class,
```

Comment:
imo, no need to register here, let user enable manually if needed, add phpstan notice to `ignoreErrors` config, see example

https://github.com/rectorphp/rector-src/blob/295bf3e51d7577a81633737766ad0a6c20e00a3f/phpstan.neon#L219C23-L219C34

Reply:
Ok